buildenv/jenkins/JenkinsfileBase

```diff
@@ -158,43 +142,72 @@ def runTest() {
 					env.DISPLAY = "${DISPLAY}"
 					echo "env.DISPLAY is ${env.DISPLAY}"
 					echo 'Running tests...'
-					sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _$TARGET"
+					sh "$WORKSPACE/openjdk-tests/maketest.sh $WORKSPACE/openjdk-tests/$subDir _$TARGET"
 				}
 			} else {
 				echo 'Running tests...'
-				sh "$OPENJDK_TEST/maketest.sh $OPENJDK_TEST _$TARGET"
+				sh "$WORKSPACE/openjdk-tests/maketest.sh $WORKSPACE/openjdk-tests/$subDir _$TARGET"
 			}
 		}
 	}
 }
 
-def post() {
+def post(test_target) {
 	stage('Post') {
 		timestamps{
 			step([$class: "TapPublisher", testResults: "**/*.tap"])
 			junit allowEmptyResults: true, keepLongStdio: true, testResults: '**/work/**/*.jtr.xml, **/junitreports/**/*.xml'
 			if (currentBuild.result == 'UNSTABLE') {
-				archiveArtifacts artifacts: '**/*.tap', fingerprint: true, allowEmptyArchive: true
 				archiveArtifacts artifacts: '**/work/**/*.jtr, **/junitreports/**/*.xml', fingerprint: true, allowEmptyArchive: true
-				if (params.TARGET == 'system') {
-					sh 'tar -zcf openjdk-systemtest-results.tar.gz $WORKSPACE/openjdk-tests/TestConfig/test_output_*'
-					archiveArtifacts artifacts: '**/openjdk-systemtest-results.tar.gz', fingerprint: true, allowEmptyArchive: true
-				}
-				if (env.BUILD_LIST == 'jck') {
-					sh 'tar -zcf jck-test-results.tar.gz $WORKSPACE/openjdk-tests/TestConfig/test_output_*'
-					archiveArtifacts artifacts: '**/jck-test-results.tar.gz', fingerprint: true, allowEmptyArchive: true
-				}
+				sh "tar -zcf $test_target-test-results.tar.gz $WORKSPACE/openjdk-tests/TestConfig/test_output_*"
```

Comment:
Suggest using ${test_target}-test-results.tar.gz, which is more readable.

Reply:
If we are going to change the name, I think it should be something like `${test_target}_test_output.tar.gz`.
`test-results` is confusing.

Reply:
updated in the new change set